files/templates/contact.html

```diff
@@ -23,6 +23,10 @@ <h1 class="article-title">Contact {{SITE_TITLE}} Admins</h1>
 
 	<p>Use this form to contact {{SITE_TITLE}} Admins.</p>
 	<form id="contactform" action="/send_admin" method="post" enctype="multipart/form-data">
+		{% if not v %}
+		<label class="mt-3">Your Email</label>
+		<input autocomplete="off" class="form-control" name="email" value="">
```

Comment:
Maybe this should have some extra text, something to the effect of, email is optional, but we won't have any way to respond to you without it

Reply:
Good call! Done.